rebasehelper/build_helper.py

```diff
@@ -283,7 +283,7 @@ class MockBuildTool(BuildToolBase):
     logs = []
 
     @classmethod
-    def _build_rpm(cls, srpm, results_dir, root=None, arch=None):
+    def _build_rpm(cls, srpm, results_dir, root=None, arch=None, enable_option=None, **kwargs):
```

Comment:
Well my though was this. Method build is called with dictionary which represents some of method parameters. My though was that in future you will forget to add enable_option into this dict and you will end up with error. 

I can remove it and check it before the call but my second though was that in future we may use **kwargs in _build_rpm method.